init.sql

```diff
@@ -1,77 +1,90 @@
-CREATE TABLE IF NOT EXISTS public."Users"
-(
-    uid integer NOT NULL,
-    name character varying NOT NULL,
-    password character varying NOT NULL,
-    phonenum character varying NOT NULL,
-    address character varying NOT NULL,
-    PRIMARY KEY (uid)
+CREATE TABLE "Users" (
+  "id" bigserial PRIMARY KEY,
+  "name" varchar NOT NULL,
+  "password" varchar NOT NULL,
+  "phonenum" varchar NOT NULL,
+  "address" varchar NOT NULL
 );
 
-CREATE TABLE IF NOT EXISTS public."Orders"
-(
-    card_id integer NOT NULL,
-    user_id integer NOT NULL,
-    created_date timestamp without time zone NOT NULL,
-    total_price integer NOT NULL,
-    cart_status character varying NOT NULL,
-    shipping_address character varying NOT NULL,
-    shipping_fee integer NOT NULL,
-    voucher_id integer NOT NULL,
-    payment_info character varying NOT NULL,
-    PRIMARY KEY (card_id)
+CREATE TABLE "Orders" (
+  "id" bigserial PRIMARY KEY,
+  "user_id" bigint NOT NULL,
+  "created_at" timestamptz NOT NULL DEFAULT 'now()',
+  "total_price" integer NOT NULL,
+  "cart_status" varchar NOT NULL,
+  "shipping_address" varchar NOT NULL,
+  "shipping_fee" integer NOT NULL,
+  "voucher_id" bigserial NOT NULL,
+  "payment_info" varchar NOT NULL
 );
 
-CREATE TABLE IF NOT EXISTS public."Vouchers"
-(
-    voucher_id integer NOT NULL,
-    condition integer,
-    value integer NOT NULL,
-    date_expire timestamp without time zone,
-    PRIMARY KEY (voucher_id)
+CREATE TABLE "Vouchers" (
+  "id" bigserial PRIMARY KEY,
+  "title" varchar NOT NULL,
+  "condition" integer NOT NULL DEFAULT 0,
+  "value" integer NOT NULL,
+  "date_expire" timestamptz NOT NULL
 );
 
-CREATE TABLE IF NOT EXISTS public."Products"
-(
-    product_id integer NOT NULL,
-    p_name character varying NOT NULL,
-    category_id integer NOT NULL,
-    description character varying,
-    status character varying,
-    price integer NOT NULL,
-    PRIMARY KEY (product_id)
+CREATE TABLE "Products" (
+  "id" bigserial PRIMARY KEY,
+  "name" varchar NOT NULL,
+  "category_id" integer NOT NULL,
+  "description" varchar,
+  "status" varchar NOT NULL DEFAULT 'PENDING',
```

Comment:
Here we maybe can create ENUM for status with possible values "PENDING", "DELIVERED" by this we will guarantee we can have only those two values in the status. But whatever it's not  that important